api/app/src/external/commonwell/link/shared.ts

```diff
@@ -0,0 +1,42 @@
+import { Person } from "@metriport/commonwell-sdk";
+import {
+  Patient,
+  PatientData,
+  PatientExternalData,
+  PatientExternalDataEntry,
+} from "../../../models/medical/patient";
+import { PatientDataCommonwell } from "../patient-shared";
+import { MedicalDataSource } from "../..";
+import { filterTruthy } from "../../../shared/filter-map-utils";
+
+export const createPatientLink = (personId: string, patientId: string): string => {
+  return `/v1/person/${personId}/patientLink/${patientId}/`;
+};
```

Comment:
Let's move this to CW SDK, so clients of the SDK can pass IDs and not have to worry about building these URIs?